### ISSUE TITLE
Speed test about the argument '--fuse'?

### ISSUE THREAD

**Issue by Joejwu** [Megvii-BaseDetection/YOLOX#666]
Hello!
I removed the "--fuse" parameter when running speed tests.
The results of the test show that approximately 2ms of time will be added.
So WHAT I want to know is whether Conv and BN are also fused in the speed test of YoloV5 in your paper?

command:  python tools/eval.py -n  yolox-s -c YOLOX_outputs0/yolox_s/best_ckpt.pth -b 1 -d 1 --conf 0.001 --fp16

results:
2021-09-08 10:02:44 | INFO     | __main__:187 - 
Average forward time: 9.88 ms, Average NMS time: 1.37 ms, Average inference time: 11.24 ms
 Average Precision  (AP) @[ IoU=0.50:0.95 | area=   all | maxDets=100 ] = 0.403
 Average Precision  (AP) @[ IoU=0.50      | area=   all | maxDets=100 ] = 0.590
 Average Precision  (AP) @[ IoU=0.75      | area=   all | maxDets=100 ] = 0.439
 Average Precision  (AP) @[ IoU=0.50:0.95 | area= small | maxDets=100 ] = 0.236
 Average Precision  (AP) @[ IoU=0.50:0.95 | area=medium | maxDets=100 ] = 0.448
 Average Precision  (AP) @[ IoU=0.50:0.95 | area= large | maxDets=100 ] = 0.535
 Average Recall     (AR) @[ IoU=0.50:0.95 | area=   all | maxDets=  1 ] = 0.323
 Average Recall     (AR) @[ IoU=0.50:0.95 | area=   all | maxDets= 10 ] = 0.532
 Average Recall     (AR) @[ IoU=0.50:0.95 | area=   all | maxDets=100 ] = 0.574
 Average Recall     (AR) @[ IoU=0.50:0.95 | area= small | maxDets=100 ] = 0.388
 Average Recall     (AR) @[ IoU=0.50:0.95 | area=medium | maxDets=100 ] = 0.633
 Average Recall     (AR) @[ IoU=0.50:0.95 | area= large | maxDets=100 ] = 0.720

**Comment by ruinmessi** [comment]
Yes, we report the results of the speed test with the same settings.

**Comment by Joejwu** [comment]
> Yes, we report the results of the speed test with the same settings.

THX!
